packages/proto-loader/src/index.ts

```diff
@@ -368,6 +379,38 @@ export function loadSync(
   return createPackageDefinition(root, options!);
 }
 
+export async function loadFileDescriptorSet(
```

Comment:
None of the operations in this function are async, so the function itself does not need to be async. I would prefer to make it synchronous in that case for flexibility.